CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,40 @@
+# Code of Conduct
+
+## Conduct
+
+Contact: a member of the [COC team](https://github.com/orgs/web-platform-tests/teams/coc) (also referenced as moderator below), or a member of the community you feel you can trust.
```

Comment:
You need to list these people. I can't see this team. Also, even if I could see this team, how could I contact them privately?

Reply:
assuming we find the idea of using GH team workable, we'll need to create the team. We shouldn't list them in this file imho. Re contact, we could provide and add an email alias. I'm open to alternatives. @sideshowbarker already found one person for the team already.

Reply:
I don't think a GitHub team works as not everyone is able to see them. You need to be a member of the organization to see teams.

Reply:
How about having the list of individuals in a file called moderators.md with contact info, and link to that?

Reply:
That works.

Reply:
I realized creating a separate file seemed awkward, so I figured we might as well put the list in the CODE_OF_CONDUCT.md itself. One issue is that the GH list of WPT owners isn't public.
@foolip ? is there a public document listing the WPT owners somewhere we can point to? If not, it seems missing and we could have that under docs/ somewhere. wdyt?

Reply:
Is this repo-level admins or org-level owners? We discussed on IRC having a meta repo with things like this at the org-level. To currently list all the people who have admin access to the repo would be a very long list.

Reply:
Resolved by https://github.com/web-platform-tests/wpt/pull/18950. Thoughts on that can still be expressed in this PR or the RFC to establish the CoC however.